README.md

```diff
@@ -1,12 +1,12 @@
-# Template Repository for Distroless Images
+# Template Repository for Chainguard Images
 
-This repository contains some basic files to spin up a new distroless image.
+This repository contains some basic files to spin up a new Chainguard image.
 
 ## Image Configuration
 
-To add a new image distribution to the distroless org, create its repository
+To add a new image distribution to the chainguard-images org, create its repository
 by clicking on the
-"[Use this template button](https://github.com/distroless/template/generate)".
+"[Use this template button](https://github.com/chainguard-images/template/generate)".
 This will create a new repository using the files contained here.
 
 Modify the sample `apko.yaml` file to define the configuration of the new image:
```

Comment:
I think we should just drop the yaml section here, since it's just a copy of apko.yaml, which they should actually edit.

Reply:
I think I've removed the bit you're talking about. TAL.